LLama/runtimes/build/LLamaSharp.Backend.Cpu.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package >
+  <metadata>
+    <id>LLamaSharp.Backend.Cpu</id>
+    <version>$version$</version>
+    <title>LLamaSharp.Backend.Cpu, the backend for LLamaSharp</title>
+    <authors>llama.cpp Authors</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="expression">MIT</license>
+    <icon>icon512.png</icon>
+    <projectUrl>https://github.com/SciSharp/LLamaSharp</projectUrl>
+    <description>LLamaSharp.Backend.Cpu is a backend for LLamaSharp to use with Cpu only.</description>
+    <releaseNotes></releaseNotes>
+    <copyright>Copyright 2023 The llama.cpp Authors. All rights reserved.</copyright>
+    <tags>LLamaSharp LLama LLM GPT AI CharBot SciSharp</tags>
```

Comment:
```suggestion
    <tags>LLamaSharp LLama LLM GPT AI ChatBot SciSharp</tags>
```

Reply:
@martindevans we should add to this file:
<file src="runtimes/ggml-metal.metal" target="runtimes\osx-x64\native\ggml-metal.metal" />

I cannot make changes.

Reply:
I _think_ anyone can make suggestions. I added it as a suggestion anyway just in case :)